### PR TITLE
update ui-framework to 2.3.1

### PR DESCRIPTION
**Pull request by MarkLark86**
The changes in the ui-framework are needed for Planning 2.1

### CODE REVIEW

**Comment by tomaskikutis** [comment]
It should also be updated for the [dateTimeField extension](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/extensions/datetimeField/package.json#L17)

**Comment by MarkLark86** [comment]
I had to add `webdriver-manager` back to `devDependencies` as e2e tests were failing to run, with the exception being raised here: https://github.com/superdesk/superdesk-client-core/blob/develop/run-end-to-end-tests.js#L6

**Review by petrjasek — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -152,7 +152,8 @@
     "react-test-renderer": "^16.13.1",
     "request": "^2.88.2",
     "superdesk-code-style": "1.5.0",
-    "typescript-eslint-parser": "^18.0.0"
+    "typescript-eslint-parser": "^18.0.0",
+    "webdriver-manager": "12.1.7"
```
Comment (tomaskikutis) [inline package.json:156]: Why are you adding "webdriver-manager"? I see it was removed not long ago in #3756.

Comment (tomaskikutis) [reply package.json:156]: Never mind, I see you already added a comment in the PR regarding this.

**Pull request merged**